pkg/synopsysctl/cmd_migrate.go

```diff
@@ -771,9 +771,94 @@ func addNameLabels(namespace string, name string, appName string) error {
 	return nil
 }
 
+// migrateCleanupCmd cleanup the unused resources
+var migrateCleanupCmd = &cobra.Command{
+	Use:           "cleanup",
+	Example:       "synopsysctl migrate cleanup <from>\nsynopsysctl migrate cleanup 2019.4.2\nsynopsysctl migrate cleanup <from> -n <namespace>",
+	Short:         "Cleanup the unused resources that are created by Synopsys Operator after verification",
+	SilenceUsage:  true,
+	SilenceErrors: true,
+	Args: func(cmd *cobra.Command, args []string) error {
+		// Check number of arguments
+		if len(args) != 1 {
+			cmd.Help()
+			return fmt.Errorf("this command takes 1 argument")
+		}
+		return nil
+	},
+	RunE: func(cmd *cobra.Command, args []string) error {
+		if len(namespace) > 0 {
+			return cleanup(namespace)
+		}
+
+		// get operator namespace
+		isClusterScoped := util.GetClusterScope(apiExtensionClient)
+		if isClusterScoped {
+			namespaces, err := util.GetOperatorNamespace(kubeClient, metav1.NamespaceAll)
+			if err != nil {
+				return err
+			}
+
+			if len(namespaces) > 1 {
+				return fmt.Errorf("more than 1 Synopsys Operator found in your cluster. please pass the namespace of the Synopsys Operator that you want to migrate")
+			}
+			return cleanup(namespaces[0])
+		}
+
+		log.Errorf("namespace of the Synopsys Operator need to be provided")
```

Comment:
They have to have provided a namespace to get here, so should say something like:

failed to find the Synopsys Operator in namespace %s

Reply:
For the below 2 scenarios, we are throwing "namespace need to be provided" error message
For namespace scope operator, namespace is mandatory. 
For cluster scope operator, if customer doesn't provide the namespace, we will try to find any synopsys operator running, if we are unable to find it

Reply:
Ignore my original comment.  Just return the error here instead of logging it as the command should return error in error cases.

Reply:
done